images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -1,3 +1,9 @@
+function Get-Aria2Version {
+    ((cmd /c "aria2c -- version 2>&1") | Out-String) -match "aria2 version (?<version>\d+\.\d+\.\d+)" | Out-Null
```

Comment:
```suggestion
    (aria2c -v | Out-String) -match "(?<version>\d+\.{1,}\d+)" | Out-Null
```

Reply:
Unless powershell has a very unusual regex syntax, I think this suggested regex is broken?

Reply:
@jmarrec sorry, my fault, there should be brackets, fixed the suggestion